### PR TITLE
Explain various users across diagrams

### PR DESCRIPTION
**Pull request by tzhelyazkova**
Rename the generic User to Client Editor or Wikidata Editor or Repo Editor based on the context.
Add more stakeholders to the overview section in line with the overall whitebox diagram.

### CODE REVIEW

File: systems/overview/01-Introduction_and_Goals.md [hunk 1]
```diff
@@ -14,6 +14,22 @@ Some stakeholders and expectations may stay at this high level though.
 
 - Have placeholder pages be automatically generated for topics Wikidata contains data for, in order to gain more readers and thus more editors.
 
+### Query User
```
Comment (addshore) [inline systems/overview/01-Introduction_and_Goals.md:17]: User here but user below

**Review by addshore — APPROVED** [review]

File: systems/overview/01-Introduction_and_Goals.md (hunk 1 above)
Comment (tzhelyazkova) [inline systems/overview/01-Introduction_and_Goals.md:17]:
```suggestion
### Query user
```

**Review by Silvan-WMDE — CHANGES_REQUESTED** [review]
While "User" may have been to general, I think the users in the diagrams may now be too specific:

The Client introduction mentions three stakeholders: Client editor, Wikimedia editor and Wikipedia editor - yet the new client diagrams only show the Client editor. Shouldn't they all show up somewhere?

In the Repo Introduction, there is the Repo editor and the Wikidata editor. I'm not even sure what the difference is between those two, but the diagrams don't show, either.

I also noticed that the tables below the context diagrams still describe the now renamed "User", both in Repo and Client. If the three different stakeholders have different roles in the diagram, maybe each of them deserves a separate row? If instead they all have essentially the same role, maybe it makes sense to stick with "User" in the "Neighbour" column (and the diagram), but then list the different stakeholders in the "Description" column and link to the corresponding Stakeholders section from there?

Interestingly, in the Overall building block diagram there's a "Wikidata Editor / Viewer" and a "Client Editor / Viewer", which I like and maybe that's a an elegant solution for the diagrams here, too. Unfortunately, in the Stakeholders section we don't even have a "Wikidata Viewer" or "Client Viewer". Maybe they should be added, if we have them in the diagram? Or rename them in the diagram instead?

**Comment by tzhelyazkova** [comment]
@addshore @jakobw @Silvan-WMDE the overview diagram mentions Viewers but we don't have them defined as stakeholders anywhere.
Should we remove them from the diagram or add Readers/Viewers to the stakeholders section? If we decide to add them, which sections should we choose - client, repo or overview?

**Comment by Silvan-WMDE** [comment]
I believe the "Viewers" are a big and important group of stakeholders and I think we should take some time to define and write down which viewer groups we actually have: Client Viewers? Repo Viewers? Or rather Wikipedia Viewers and Wikidata Viewers?

If others agree, we could make this a phabricator ticket.

**Review by Silvan-WMDE — APPROVED** [review]

**Comment by tzhelyazkova** [comment]
> I believe the "Viewers" are a big and important group of stakeholders and I think we should take some time to define and write down which viewer groups we actually have: Client Viewers? Repo Viewers? Or rather Wikipedia Viewers and Wikidata Viewers?
> 
> If others agree, we could make this a phabricator ticket.

Filed [T273973](https://phabricator.wikimedia.org/T273973)

**Pull request merged**